site/source/tutorials/accessibility.md

```diff
@@ -0,0 +1,250 @@
+
+___
+
+### TabController
+
+TabController creates a set of tab buttons that control the content of a container. The tab buttons use `role="tab"` and are associated with their content through `aria-controls` and `aria-labelledby`. Only one tab button is in the tab order at a time, and arrow keys are used to switch between tab buttons. This approach both directly follows the [ARIA Authoring Practice Guidelines](https://www.w3.org/TR/wai-aria-practices-1.1/) and avoids polluting the tab order with a large number of tab stops in the case of a single TabController with many tabs.
+
+##### Keyboard Interaction
+- **Left arrow**: Moves to previous tab button, wrapping from first to last
+- **Right arrow**: Moves to next tab button, wrapping from last to first
+- **Down arrow**: Moves to next tab button only on vertically oriented tabs
+- **Up arrow**: Moves to previous tab button only on vertically oriented tabs
+- **Page Up**: Moves to first tab button
+- **Page Down**: Moves to last tab button
+- **Escape**: Closes focused tab button if it is closeable
+
+___
+
+### TitlePane
+
+TitlePane creates an accordion widget, which is functionally a button that expands a content container. Collapsed content is obscured with `aria-hidden`, and the button and content container are associated with `aria-controls` and `aria-labelledby`.
+
+##### A11y properties
+- `headingLevel`: Optionally customize the heading level of the button controlling the accordion.
+___
+
+This overview only touched on properties that primarily exist for accessibility-related reasons, but any design decision or property change will end up affecting accessibility in some way. For some of those, such as `invalid`, `disabled`, and `readOnly`, we handle setting ARIA attributes in the background in addition to toggling classes without any extra attention needed from the widget author. In other cases, such as `getResultLabel` in ComboBox or `renderMonthLabel` in Calendar, it is entirely up to the author to ensure the returned result includes clear, screen-reader-accessible text content.
+
+## Styling
+The base Dojo 2 theme meets WCAG AA color contrast guidelines, but it is up to the widget author to double check the final styles as changes to font size or background could affect readability. Widget CSS packaged with Dojo2 also includes `:focus` styles for all focusable nodes.
+
+There is a set of base styles provided by `@dojo/widgets` separate from themes, containing basic utility classes like `.visuallyHidden`, which will make content invisible to sighted users but still allow it to be read by screen readers. To use it, import `baseCss` separately like so:
+
+```js
+import * as css from ‘path/to/your/css’;
+import * as baseCss from ‘@dojo/widgets/common/styles/base.m.css’;
+
+@theme(css)
+class MyWidget extends WidgetBase {
+	render() {
+		return v(‘div’, {
+			classes: this.classes().fixed(baseCss.visuallyHidden)
+		}, [ ‘Screen reader instructions’ ]);
+	}
+}
+```
+
```

Comment:
Maybe worth adding a sentence referring to the full theming tutorial that is being worked on with a coming soon link for now?